src/components/cards/CategoryCard.jsx

```diff
@@ -0,0 +1,10 @@
+import { Link } from "react-router-dom";
+import "./cards.css";
+export function CategoryCard({value}){
+    return (
+        <Link to="/products" className="category-card">
+            <img className="img-round img-round-sizing" src={value.image.src} alt="camera" />
```

Comment:
suggestion: you can use destructuring in function argument only.

Reply:
Yes, I have refactored this and destructured the props.